feature-libs/checkout/b2b/components/checkout-payment-type/checkout-payment-type.component.ts

```diff
@@ -108,7 +134,8 @@ export class CheckoutPaymentTypeComponent {
   constructor(
     protected checkoutPaymentTypeFacade: CheckoutPaymentTypeFacade,
     protected checkoutStepService: CheckoutStepService,
-    protected activatedRoute: ActivatedRoute
+    protected activatedRoute: ActivatedRoute,
+    protected globalMessageService: GlobalMessageService
```

Comment:
This is breaking change ... as far as I know, you need to handle it.

Reply:
here is example how to handle (launchDialogService is a newly added property in the constructor)
![image](https://github.com/SAP/spartacus/assets/11823988/45dca8f9-8c49-4047-a700-27aaadf4fec2)

Reply:
Thanks Hak, i'll check it out LaunchDialog for an error message